File: naelson022/programacao_basica/Questao08.java
import java.util.Scanner;

public class Questao08 {
    public static void main (String [] args) {
        Scanner s;
        double a;
        double b;
        double c;
        double d;
        double e;
        double f;
        double g;
        double h;
        

        s = new Scanner (System.in);
        System.out.println("** volume do cubo**");
        System.out.println(" digite o lado do cubo ");
        a = s.nextDouble ();
        b = a * a * a;
        System.out.println(" o volume do cubo sera " + b);

        System.out.println("** volume do paralelepipado**");
        System.out.println(" digite a altura do paralelepido ");
        c = s.nextDouble ();
        System.out.println(" digite o comprimento do paralelepipado");
        d = s.nextDouble ();
        System.out.println(" digite a base do parelelepido");
        e = s.nextDouble ();
        f = c * d * e;
        System.out.println(" o volume do paralelepido " + f );
        
        System.out.println("** volume das esfera **");
        System.out.println(" informe o raio da esfera : ");
        g = s.nextDouble ();
        h = 4 * 3.14 * g * g * g /3;
    }
}

File: vinicius012/programacao_basica/questao08.py
print("*** Calculadora para  Minerva ***")
a = int(input(" Digite o lado do cubo: "))
b = int(input(" Digite a altura do do paralelepido: "))
c = int(input(" Digite o comprimento do paralelepipedo: "))
d = int(input(" Digite a profundidade do paralelepipedo: "))
e = float(input(" Digite a base do cone: "))
f = float(input(" Digite a altura do cone: "))
g = float(input(" Digite o raio da esfera: "))
x = a ** 3
y = b * c * d
z = 1 / 3 * 3.141592 * e ** 2 * f
w = 4 / 3 * 3.141592 * g ** 3
print(" ")
print(" O volume do cubo e " + str(x))
print(" O volume do paralelepipedo e " + str(y))
print(" O volume do cone e " + str(z))
print(" O volume da esfera e " + str(w))

